### PR TITLE
Adds the first version of PPP

### PR DESCRIPTION
**Pull request by ErnestCuong**

### CODE REVIEW

File: docs/team/ernestcuong.md [hunk 1]
```diff
@@ -5,13 +5,42 @@ title: Cuong's Project Portfolio Page
...
-  * Developer Guide:
-    * Target user profile, value proposition, user stories, use cases, non-functional requirements, glossary
+* **Code contributed**:
+  * My contributions to the TutorAid code base can be found [here](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=W16&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=ErnestCuong&tabRepo=AY2122S1-CS2103T-W16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false).
+
+* **New features implemented**:
+  * Implemented a `progress` feature for student for v1.2.
+    * Relevant PR(s): [#64](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/64/files)
+    * Implemented the `add -p` and `del -p` commands to add and delete progress from a student.
+    * Worked closely with the `logic` component to implement the commands and the parser for adding and deleting progress, as well as `model` to add a new `Progress` field to `Student`
```
Comment (wpinrui) [inline docs/team/ernestcuong.md:19]: Maybe this should be `Logic` and `Model` (capitalise)

File: docs/team/ernestcuong.md [hunk 2]
```diff
@@ -5,13 +5,42 @@ title: Cuong's Project Portfolio Page
...
+    * Only allowed one progress entry in this iteration.
+  * Enhanced the `progress` feature so that each student can have up to 10 progress entries for v1.3.
+    * Relevant PR(s): [#105](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/105)
+    * Replaced the `Progress` field in `Student` with `ProgressList` that keeps track of up to 10 progress entries.
+    * Justification: I wanted to abstract away the management of progress entries from `Student` to preserve the single responsibility principle.
+  * Implemented the functionality of `add -sl` and `del -sl` commands that add and remove students from lessons for v1.3.
+    * Relevant PR(s): [#125](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/125)
+    * Worked closely with the `logic` component to implement the commands and the parser for adding students to and removing students from lessons, as well as `model` to add a new `LessonNames` field to `Student`.
```
Comment (wpinrui) [inline docs/team/ernestcuong.md:27]: Same here

**Review by wpinrui — APPROVED** [review]
Mostly LGTM

File: docs/team/ernestcuong.md (hunk 1 above)
Comment (ErnestCuong) [reply docs/team/ernestcuong.md]: Thanks a lot

**Review by Preshita01 — APPROVED** [review]
LGTM

**Pull request merged**